dashboard/src/api/index.ts

```diff
@@ -0,0 +1,73 @@
+import * as axios from 'axios';
+import { Moment } from 'moment';
+
+import AuthManager from './auth-manager';
+import { RegionList, RegionStatsList } from './types';
+
+interface SuccessCallback<T> {
+    (response: axios.AxiosResponse<T>): void;
```

Comment:
I guess these will be extended later? Otherwise the arguments for both Success and error handlers are unused.

Reply:
It's a callback interface. It means that the you give it a function that will be called with a response. It's not unused, see line 61 where callback is called with a response.

Reply:
Alright 👍